### ISSUE TITLE
Dataset generation issues

### ISSUE THREAD

**Issue by mjfbuse** [lbcb-sci/herro#82]
Hi, I have a few questions about the training dataset:

1. When extracting training sets, each window with NO supportive positions will be kept or dropped? 

2. For the loss function, do we just add both or consider weights on supportive positions as their frequency is quite low?

Thank you!
